bespin/collector.py

```diff
@@ -159,6 +159,7 @@ def convert_stack(path, val):
             env = configuration[["environments", environment]]
             if isinstance(env, six.string_types):
                 environment_as_dict = configuration[["environments", env]].as_dict()
+                env = configuration["environments", env]
```

Comment:
oh, I see, python changes that into a tuple for you.... I'd rather be explicit to be honest